tasks/bootstrap-firewall.yml

```diff
@@ -23,7 +22,7 @@
 - name: Open ports by IP
   ufw:
     rule: allow
-    src:  "{{ item.value.ip }}"
-    port: "{{ item.value.port }}"
+    src:  "{{ item.ip }}"
+    port: "{{ item.port }}"
     proto: tcp
-  with_dict: "{{ port_ips | default({}) }}"
+  loop: "{{ sb_debian_base_ports_ips |default([]) }}"
```

Comment:
@speedlight is this change backward compatible with the `min_ansible_version` specified in the meta configuration file?

Reply:
Yes @sestrella, the with_<loopup> is not deprecated yet, but the use of loop is recommended by ansible in order to make use of this improved version (with_ vs loop) and  avoid changes in future versions.